pkg/data/util/podsecurityadmission_util.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+var FeatureAppNS = []string{
+	"ingress-nginx",              // This is for Ingress, not feature app
```

Comment:
I'm a bit nervous about including this in a list of namespaces to clean up if this is going to work in the local cluster. The ingress-nginx release for the local cluster is going to be managed by our users - I would expect that they would need to manually migrate this release before upgrading. Do you have an idea on what the workflow looks like for this chart?

Reply:
It seems the chart for the Nginx Ingress controller has a switch for turning on/off PSPs: https://github.com/kubernetes/ingress-nginx/blob/a00cf624b7ef43f44278bf1ce1114e0de663104e/charts/ingress-nginx/templates/controller-psp.yaml#L2

I think it's fair to remove non-Rancher namespaces but then I cannot consume this list anymore, and will therefore change the code back to what it was before.

Reply:
Updated.